chainer/functions/normalization/batch_renormalization.py

```diff
@@ -173,21 +171,38 @@ def batch_renormalization(x, gamma, beta, rmax, dmax, eps=2e-5,
     individual examples rather than the entire minibatch.
 
     .. note::
+
         This function does not perform in-place update to
         ``running_mean`` and ``running_var`` by default, contrary to
         :func:`~chainer.functions.batch_normalization`.
         If the function is called, it will not be possible to access the
         updated running mean and variance statistics, because they are members
         of the function object, which cannot be accessed by the caller.
         If it is desired to update the running statistics, call the function
-        with `update_statistics=True` option.
+        with ``update_statistics=True`` option.
+
+    .. note::
+
+        For the consistency with Batch Normalization, this function
+        intentionally ignores some of the theoretical flaws in Algorithm 1 of
+        the Batch Renormalization paper:
+
+        - The function maintains the moving average of variances
```

Comment:
A nitpicking suggestion that makes the sentence more clear while reading.

```suggestion
        - ``F.batch_renormalization`` maintains the moving average of variances
```